.github/workflows/draft-pdf.yml

```
@@ -0,0 +1,28 @@
+name: Draft PDF
+on:
+    push:
+      paths:
+        - paper/**
+        - .github/workflows/draft-pdf.yml
+
+jobs:
+  paper:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)